modules/core/shared/src/main/scala/eu/timepit/refined/collection.scala

```diff
@@ -342,4 +342,7 @@ private[refined] trait CollectionInference {
 
   implicit def sizeInference[A, B](implicit p1: A ==> B): Size[A] ==> Size[B] =
     p1.adapt("sizeInference(%s)")
+
+  implicit def sizeNonEmpty[N]: Size[Interval.Closed[_1, N]] ==> NonEmpty =
```

Comment:
For consistency with the other instances:
```suggestion
  implicit def sizeNonEmptyInference[N]: Size[Interval.Closed[_1, N]] ==> NonEmpty =
```